src/FSharp.Control.TaskSeq/TaskSeq.fs

```diff
@@ -253,6 +253,10 @@ module TaskSeq =
     let chooseAsync chooser source = Internal.choose (TryPickAsync chooser) source
     let filter predicate source = Internal.filter (Predicate predicate) source
     let filterAsync predicate source = Internal.filter (PredicateAsync predicate) source
+    let takeWhile predicate source = Internal.takeWhile (*inclusive:*)false (Predicate predicate) source
+    let takeWhileAsync predicate source = Internal.takeWhile false (PredicateAsync predicate) source
+    let takeWhileInclusive predicate source = Internal.takeWhile true (Predicate predicate) source
+    let takeWhileInclusiveAsync predicate source = Internal.takeWhile true (PredicateAsync predicate) source
```

Comment:
I'll probably make the `true|false` into a two-case struct DU for clarity.

Reply:
Done, see changes.

Reply:
I should have done that, good idea